apps/settings/js/panels/improve_browser_os/panel.js

```diff
@@ -7,6 +7,9 @@ define(function(require) {
   const METRICLEVELSETTING = 'metrics.selectedMetrics.level';
   const DOGFOODSETTING = 'debug.performance_data.dogfooding';
   const SHAREDATASETTING = 'debug.performance_data.shared';
+  const DEVTOOLSOVERLAY = 'devtools.overlay';
+  const HIDEHUDWIDGETS = 'hud.hide';
```

Comment:
show/hide HUD widgets does not related to this function, we could just change its default value in common-settings.js